packages/common/src/contract/interface.ts

```diff
@@ -0,0 +1,13 @@
+import type { CellDep, Script } from '@ckb-lumos/lumos'
+
+export type ContractDeploymentInfo = {
+  name: string
+  path: string
```

Comment:
The path here should be a relative path? after all, every personal computer on the absolute path are not quite the same, if it is a relative path, then which path should be used as root?

Reply:
Absolute path maybe suitable here I think.

Reply:
> Absolute path maybe suitable here I think.

It's better to append arguments along with the conclusion